app/signin.tsx

```diff
@@ -0,0 +1,11 @@
+import { View } from 'react-native';
+import SignIn from '../components/SignIn';
+import tw from 'twrnc';
+
+export default function SignInPage() {
+  return (
+    <View style={tw`h-full bg-cyan-50`}>
```

Comment:
Nu are sens `h-full`, iar `bg-cyan-50` il poti pune direct in ScrollView-ul ce va fi aici